packages/documentation/src/stories/getting-started/components-angular/angular.stories.ts

```diff
@@ -3,7 +3,7 @@ import { BADGE } from '../../../../.storybook/constants';
 
 const meta: Meta = {
   id: '833ef689-a573-40f5-a6a6-30a999b94733',
-  title: 'Getting Started/Components-Angular',
+  title: 'Getting Started/Components for Angular',
```

Comment:
I also thought about that, but in a sense the name is similar to the package and with https://github.com/swisspost/design-system/pull/2722 it will be likely inside a `Packages` folder.

Anyway, I'm ok with both.